packages/main/src/ComboBox.ts

```diff
@@ -1224,8 +1224,9 @@ class ComboBox extends UI5Element {
 	get classes() {
 		return {
 			popover: {
-				"ui5-suggestions-popover": !this._isPhone,
-				"ui5-suggestions-popover-with-value-state-header": !this._isPhone && this.hasValueStateText,
+				"ui5-suggestions-popover": true,
+				"ui5-popover-with-value-state-header-phone": this._isPhone && this.hasValueStateText,
+				"ui5-popover-with-value-state-header": !this._isPhone && this.hasValueStateText,
```

Comment:
The styles are added on a level of popover because for Desktop the value state message is part of the popover header slot, while for mobile devices the value state message is part of the content slot. I wanted to be able to have easy control to style those 2 different usages of value state message, without breaking the current implementation for Desktop, but also have easy control how to modify with css both content and header.